### DISCUSSION TITLE
Firewall rule Isolate

### DISCUSSION THREAD

**Discussion by zod076** [celzero/rethink-app#1806] [General]
Hi, what does Isolate do under firewall rules for an app ?

**Comment by onetimecontributor** [comment]
it's when the app is allowed to access only particular whitelisted domains or IPs, from what I recall

**Reply by ignoramous** [reply]
Correct. Also see: https://www.reddit.com/r/rethinkdns/comments/11vxyp6/the_6_icons_in_apps/ (though some semantics around "Bypass Universal" has changed (in that, both IP and Domain rules are now *bypassed*, as one would expect and not just IP rules).

**Reply by xa4hf8** [reply]
I am wondering whether "Bypass Universal" is working correctly. For example, I have Bypass enabled for the platform DNS client, but I still see traffic from that application blocked in the log. When i open the Error dialog to see why, it directs me to check the "All apps tab". But there are no blocking rules for this app.

![TCP 853 blocked](https://github.com/user-attachments/assets/03f088ea-1776-4d15-aa12-b815e1d769c4)

![Error - all apps tab](https://github.com/user-attachments/assets/e466dc11-8fb9-4822-8c6b-fb9ff198ed1f)

![DNS app](https://github.com/user-attachments/assets/8cbd74e4-0b29-4df1-bc93-976658932655)

Speaking of semantics, the word "tab" briefly confused me, until i realized that it was referencing the Apps tile under the Home tab. It might be helpful to display an activity button like this which opens the correct menu:

![Error - all apps concept](https://github.com/user-attachments/assets/a2652515-438e-437b-9636-1cf125d8ac33)

**Reply by onetimecontributor** [reply]
> I am wondering whether "Bypass Universal" is working correctly. For example, I have Bypass enabled for the platform DNS client, but I still see traffic from that application blocked in the log. When i open the Error dialog to see why, it directs me to check the "All apps tab". But there are no blocking rules for this app.

have you checked definitions of those options?
check what Exclude means, also, what are those "user-defined" rules that are on your screen?
